packages/design-system-web/src/lib/importMapLibre.tsx

```diff
@@ -0,0 +1,38 @@
+import type MapLibreGL from 'maplibre-gl';
+import { isNode } from './isNode';
+
+/**
+ * Load and return plotly library from CDN in browsers.
+ * In node server environment returns null.
+ * @returns Plotly (in browser) or null (in server)
+ */
+export const importMapLibre = async () =>
+  new Promise<typeof MapLibreGL | null>((resolve, reject) => {
+    // if we are on a node server resolve with null
+    if (isNode()) return resolve(null);
+
+    // if MapLibre is already defined resolve it
+    if ('maplibregl' in window) {
+      return resolve(window.maplibregl as any); //TYPE IS NOT RIGHT! MIGHT BE A PROBLEM WITH THE IMPORT. PLEASE FIX!
+    }
+
+    // create new script element
+    const script = document.createElement('script');
+
+    // register on load callback and resolve MapLibre
+    script.onload = () => resolve(window.maplibregl as any);
+
+    // register on error callback and reject
+    script.onerror = reject;
+
+    // set url
+    script.src = 'https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js';
+
```

Comment:
Fix the version to avoid errors when breaking changes are introduced in future releases.

```suggestion
    script.src = 'https://unpkg.com/maplibre-gl@3.4.1/dist/maplibre-gl.js';
```